.test-infra/validate-runner/build.gradle

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin : "application"
+
+group 'org.apache.beam'
+description = "Apache Beam :: Validate :: Runner"
+
+repositories {
+    mavenCentral()
+//    maven {
+//        url "https://repo.spring.io/libs-release/"
+//    }
+    maven {
+        url "https://repo.jenkins-ci.org/releases/"
+    }
+    maven {
+        url "https://jcenter.bintray.com/"
+    }
+}
+
+dependencies {
+    compile group: 'com.cdancy', name: 'jenkins-rest', version: '0.0.22'
+    compile 'com.offbytwo.jenkins:jenkins-client:0.3.8'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.0.1'
+    compile group: 'org.jenkins-ci.plugins', name: 'junit', version: '1.34'
+    compile group: 'org.yaml', name: 'snakeyaml', version: '1.8'
+    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.11.3'
+    compile 'org.slf4j:slf4j-simple:1.7.9'
+    compile group: 'junit', name: 'junit', version: '4.12'
+    compile group: 'org.apache.beam', name: 'beam-sdks-java-core', version: '2.26.0'
+    compile group: 'org.apache.beam', name: 'beam-sdks-java-core', version: '2.26.0', classifier: 'tests'
+    compile group: 'org.apache.beam', name: 'beam-runners-flink_2.11', version: '2.16.0'
+    compile group: 'org.apache.beam', name: 'beam-runners-flink_2.11', version: '2.16.0', classifier: 'tests'
+    compile group: 'org.apache.beam', name: 'beam-runners-spark', version: '2.28.0'
+    compile group: 'org.apache.beam', name: 'beam-runners-spark', version: '2.28.0', classifier: 'tests'
+    compile group: 'org.apache.beam', name: 'beam-runners-core-java', version: '2.26.0'
+    compile group: 'org.apache.beam', name: 'beam-runners-core-java', version: '2.26.0', classifier: 'tests'
```

Comment:
I wonder if it's possible to depend on Beam's current branch instead of an older version. Could you try something like this:
```
compile project(path: ":sdks:java:core").sourceSets.test.output
```

or perhaps, although this may not work:
```
compile project(path: ":sdks:java:core") classifier: 'tests'
```

Let me know if that works or not. If that does not work, then at least we should choose a single Beam version, set a single variable, and use that variable in all these lines.